README.md

```diff
@@ -1 +1,187 @@
 # Work In Progress - Onfido Java Library
+
+The official Java library for integrating with the Onfido API.
```

Comment:
Minor: Could we put a "warning" emoji somewhere round here, just because I wouldn't be surprised if some people totally miss the title.